app/models/ext_management_system.rb

```diff
@@ -27,6 +27,9 @@ def self.supported_types_and_descriptions_hash
 
   belongs_to :provider
   belongs_to :tenant
+  include ComplianceMixin
+  include CustomAttributeMixin
+  after_initialize :load_custom_attributes
```

Comment:
@alongoldboim so this https://github.com/ManageIQ/manageiq/pull/11369 needs to be merged first ?